### ISSUE TITLE
Inconsistencies with Glaives kills in the api

### ISSUE THREAD

**Issue by FubyCoder** [Bungie-net/api#1644]
I recently played a lot of crucible matches with a glavie equipped , and i noticed that a lot of kills with glaives are counting as melee kills

It can be noticed here :

https://www.bungie.net/it/PGCR/10588164688
https://www.bungie.net/it/PGCR/10588041568

Looks like the only glaives kill counted correctly are the one using the shooting mode of the glaives

**Comment by justrealmilk** [comment]
I suspect this happens higher up the chain than at the PGCR or API level (game bug?).

**Comment by FubyCoder** [comment]
> I suspect this happens higher up the chain than at the PGCR or API level (game bug?).

It could be an in game bug, since kills with glaive (using the melee option) can activate some melee perks of the void subclass (like the one that provides health regen from melee kills)

**Comment by jshaffstall-bng** [comment]
Yeah this is known behavior on the game-side implementation of Glaive melee kills and activity stats.

Don't know if I'd characterize it as a bug, but I understand that it's not exactly in-line with expectations.

This is how we should expect it to stay for now.
